db/migrations/20210415144236-social_graph.sql

```diff
@@ -0,0 +1,16 @@
+-- +migrate Up
+CREATE TABLE social_graph (
+  id bigserial PRIMARY KEY,
+  amount bigint NOT NULL,
+  buyer_address text NOT NULL,
+  creator_address text NOT NULL,
```

Comment:
can we add an index on these fields ?

Reply:
How's this: `CREATE INDEX relationship ON social_graph (buyer_address, creator_address);`

Could make it unique, but if they switch validators, there will be multiple entries...

Reply:
Its fine Its mainly because this will be the main query path used in the graphql layer so should help speeding things